k8s/templates/frontend-pgbouncer.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: v1
```

Comment:
I'm trying out this style of having related manifests be in a single YAML file, but I'm unsure if it's helpful or not. What do you think @lyw07?

Reply:
I think it looks good here. it might be a bit concerning in the future if the yaml file gets super long. but for this pgbouncer one, I think that's not an issue.

Reply:
Agree! Let's refactor if this gets longer than a few hundred lines.